lls/main.c

```diff
@@ -1080,30 +1082,47 @@ assign_lls_queue_ids(struct lls_config *lls_conf)
 	 */
 
 	if (lls_conf->net->front.rss) {
+		lls_conf->mp = create_pktmbuf_pool("lls",
+			lls_conf->lcore_id, num_mbuf);
+		if (lls_conf->mp == NULL) {
+			ret = -1;
+			goto fail;
+		}
+
```

Comment:
The mbuf pool of LLS is not optional because of the calls to `rte_pktmbuf_alloc()` in its code. But the size of the pool may vary depending on hardware support.